src/main/target/common_fc_post.h

```diff
@@ -96,6 +96,9 @@
 #if !defined(USE_DSHOT_DMA) && !defined(USE_DSHOT_DMAR)
```

Comment:
I think since this is now flexible, and allows users to enable DMAR and test with it on targets that do not have it tested and enabled by default, we can remove `USE_DSHOT_DMA`, and replace `USE_DSHOT_DMAR` with `ENABLE_DSHOT_DMAR true`, and then define `USE_DSHOT_DMAR` in `common_fc_pre.h`, and set  `ENABLE_DSHOT_DMAR  false` here for targets that define nothing.

Reply:
I'm a bit confused. Can you give me code snippets for pre and post?

Reply:
`common_fc_pre.h`:

(in the section for the minimum flash size supporting it)
```
#define USE_DSHOT_DMAR
```

`common_fc_post.h`:

```
#ifndef ENABLE_DSHOT_DMAR
￼#define ENABLE_DSHOT_DMAR false
#endif
```